### DISCUSSION TITLE
What folder do models go into

### DISCUSSION THREAD

**Discussion by ghost** [oobabooga/text-generation-webui#232] [General]
The directions say "Models should be placed under models/model-name. For instance, models/gpt-j-6B for [GPT-J 6B]"

there is a folder for Models, and a subfolder model-names... does it go in models or model-name...instructions arent clear :(

Also, does this oobabooga folder go inside the text-generation-webui folder, or are they kept seperate, or vice versa?

and does the conda folder need to be in either of the oobagooba folder or the text-generation-webui folder, or niether of them?

I get this error too

 EnvironmentNameNotFound: Could not find conda environment: textgen
You can list all discoverable environments with `conda info --envs`.

**Comment by jtk1996** [comment]
Just ran into the same problem.

Solution: Models should be placed under oobabooga\text-generation-webui\models

The models will be stored there automatically when you use the download-model.bat

Unfortunately I cannot help you with the conda problem.

Edit:Just found a helpful Nerdy-Rodent Youtube video where he explains the installation: https://www.youtube.com/watch?v=rGsnkkzV2_o

**Reply by ghost** [reply]
Thank you, I appreciate it alot.

sometimes solving one problem, leads to a new problem, that helps solve the other problem lmao...
